ci-operator/step-registry/openstack/gather/openstack-gather-commands.sh

```diff
@@ -34,7 +36,8 @@ collect_bootstrap_logs() {
 				FIPS+=("${IP}")
 				openstack server add floating ip ${CLUSTER_ID}-bootstrap ${IP}
 			else
-				IP=$(openstack server show ${BOOTSTRAP_NODE} --colum addresses --format json | jq -r '.addresses | .[][0]')
+				ADDRESSES=$(openstack server show "${BOOTSTRAP_NODE}" --column addresses --format json)
+				IP=$(echo "${ADDRESSES}" | jq -r '.addresses[][0]')
```

Comment:
I think this will still cause the issue because the addresses field contain the name of the network plus the address, like `restricted_network=172.16.0.240`. Perhaps it would be needed to treat the string to eliminate the name of the network?

Reply:
The `jq` command is supposed to handle that for us.

Reply:
@MaysaMacedo you were right, the version of openstack client used in CI had a different output than what I tested with, which caused it to fail.
I'm fixing this with https://github.com/openshift/release/pull/33166.